src/views/GameView.vue

```diff
@@ -33,25 +32,25 @@ function nextCode() {
     currentCode.value = game.getCurrentCode();
     showNextButton.value = false;
     hasNextCode.value = game.hasNextCode();
-    rightWordId.value = wrongWordId.value = -1;
+    feedbackSolution.value = [];
   } else {
     emit('endGame');
   }
 }
 
-function submitSolution(selectedWordId: number) {
-  const correct: boolean = game.submitWrongCode(selectedWordId);
+function submitSolution(selectedBugs: ISolution) {
+  const feedback: Array<boolean> = game.submitWrongCode(selectedBugs);
   chatHistory.value.push({ from: ChatParticipant.ME, message: 'I think I found the bug. Is the programm now running?' });
   chatHistory.value.push({ from: ChatParticipant.OTHER, message: '...' });
   setTimeout(() => {
     chatHistory.value.pop();
-    if (correct) {
+    if (feedback.find((bool) => bool == false) == null) {
```

Comment:
I've seen this call at least twice now.
I still don't have an idea what it does.
Time for a function?

Reply:
created a function named `passedCurrentCode()` in `BugFinderGame`